workspaces/jenkins/app-config.yaml

```diff
@@ -33,7 +43,13 @@ techdocs:
 
 auth:
   providers:
-    guest: {}
+    github:
```

Comment:
We prefer to use Guest auth for these examples apps, let's keep that in and skip the GitHub part. 👍